hack/make/provision.mk

```diff
@@ -23,7 +23,8 @@ GIT_COMMIT_SHA=$(shell git rev-parse --short=8 HEAD)
 HIBERNATION_HOUR=$(shell echo $$(( ( $(shell date +%H | sed s/^0//g) + 5 ) % 24 )))
 GARDENER_K8S_VERSION ?= $(ENV_GARDENER_K8S_VERSION)
 GARDENER_OS_VERSION ?= $(ENV_GARDENER_OS_VERSION)
-GARDENER_CLUSTER_NAME=$(shell echo "ci-${GIT_COMMIT_SHA}-${GARDENER_K8S_VERSION}" | sed 's/\.//g')
+# Cluster name is also set via load test. If its set then use that else use ci-XX
+GARDENER_CLUSTER_NAME ?= $(shell echo "ci-${GIT_COMMIT_SHA}-${GARDENER_K8S_VERSION}" | sed 's/\.//g')
```

Comment:
Maybe use a simple name by default (like "test") and move the ci specific things into the CI. So maybe we should have indeed a shell script for integration test and for load test which determines all the gardener settings specific to the scenario. And in the make file we have a very simple setup, so that we even don't need the commit sha here for example. What do you think?

Reply:
I will handle it in separate PR